api/src/pcapi/routes/backoffice_v3/templates/accounts/get/details/bookings.html

```diff
@@ -68,6 +68,10 @@
                 <span class="fw-bold">Crédit utilisé par le jeune :</span>
                 {{ booking.deposit.type | format_deposit_type | safe }}
               </p>
+              <p class="mb-1">
+                <span class="fw-bold">Email utilisé par la réservation :</span>
```

Comment:
```suggestion
                <span class="fw-bold">Email à la réservation :</span>
```
(plus court et plutôt `à` que `par`)